src/heads-up/views/app-link.vue

```diff
@@ -0,0 +1,245 @@
+          class="app-link__list"
+        >
+          <li
+            v-for="(item, index) in section"
+            :key="index"
+            class="app-link__list-item"
+          >
+            <span
+              v-if="item.length === 1"
+              class="app-link__attribute-value"
+            >
+              <span class="app-link__attribute">{{ item[0][0] }}</span> <span
+                class="app-link__value"
+                :title="item[0][1]"
+              >{{ item[0][1] }}</span>
+            </span>
+            <details
+              v-else
+              class="app-link__property-table"
+            >
+              <summary>
+                <ChevronRightIcon />
+                <span class="app-link__attribute-value">
+                  <span class="app-link__attribute">{{ item[0][0] }}</span> <span
+                    class="app-link__value"
+                    :title="item[0][1]"
+                  >{{ item[0][1] }}</span>
+                </span>
+              </summary>
+
+              <dl class="app-link__table-row app-link__attribute-value">
+                <template v-for="(row, rowIndex) in item">
+                  <template v-for="(column, i) in row">
+                    <!-- eslint-disable-next-line vue/no-v-html, vue/max-attributes-per-line -->
+                    <component
+                      :is="i === 0 ? 'dt' : 'dd'"
+                      :key="`${row}${column}`"
+                      :class="{
+                        'app-link__attribute': i === 0,
+                        'app-link__value': i === 1
+                      }"
+                      :title="i === 1 ? column : undefined"
+                      :hidden="rowIndex === 0"
+                    >
+                      {{ column }}
+                    </component>
+                  </template>
+                </template>
+              </dl>
+            </details>
+          </li>
+        </ul>
+      </details>
+    </panel-section>
+
+    <panel-section title="Resources">
+      <resource-list>
+        <li>
+          <external-link href="https://htmlhead.dev/">
+            Guide to HTML5 <code>&lt;head&gt;</code> elements
+          </external-link>
+        </li>
+      </resource-list>
+    </panel-section>
+  </div>
+</template>
+
+<script>
+  import { mapState } from 'vuex'
+  import { ExternalLink, PanelSection, ResourceList } from '../components'
+  import schema  from '../lib/schemas/app-meta-schema'
+  import ChevronRightIcon from '../assets/icons/chevron-right.svg'
+
+  export default {
+    components: { ExternalLink, PanelSection, ResourceList, ChevronRightIcon },
+    data: () => ({
+      includeData: false
+    }),
+    computed: {
+      ...mapState(['head']),
+      appLinkData() {
+        return this.head.link
+          .filter(item => item.rel)
+          .map(item => {
+            if (this.includeData) return item
+            return Object.keys(item)
```

Comment:
What is the idea behind this? If I remove this piece of code with just `return item` it looks like it does the same. I never appear to spot a key in the `item` object which has `data-` in it's name. If this is not used, you can remove the `map` altogether

Reply:
if you test this on a nuxt application, there are quite some data- attributes there. Check loavies.com for example. And since `data-` attributes do not have any specific meaning, they can be safely removed because the clutter up the interface

Reply:
we might introduce a toggle though, showing `data-` attributes for when users intentionally add them themselves

Reply:
Makes sense!